src/Timber.php

```diff
@@ -1056,6 +1056,39 @@ public static function get_pages_menu(array $args = [])
         return $menu;
     }
 
+    /**
+     * Get the navigation menu location assigned to the given menu.
+     *
+     * @param  WP_Term|int $term The menu to find; either a WP_Term object or a Term ID.
+     * @return string|null
```

Comment:
We need to mark these new methods as @api functions to end up in the docs! Also probably a good idea to add the `@since 2.3.0` tag on both methods.

Reply:
Good point! Done.